### PR TITLE
[wasm][debugger] Reusing buffer to avoid allocate every debugger message

### PR DESCRIPTION
**Pull request by thaystg**
This will speed up debugger and experience and will avoid leaking memory.

### CODE REVIEW

**Comment by thaystg** [comment]
@pavelsavara I'll wait for your approval to backport :)

**Review by radical — APPROVED** [review]

File: src/mono/wasm/runtime/library_mono.js
Comment (pavelsavara) [inline src/mono/wasm/runtime/library_mono.js:608]: perhaps allocate 125%, so that possibly you don't have to re-allocate it next time. Otherwise approved :)

**Review by pavelsavara — APPROVED** [review]

**Review by radekdoulik — APPROVED** [review]

File: src/mono/wasm/runtime/library_mono.js
Comment (thaystg) [reply src/mono/wasm/runtime/library_mono.js:608]: I didn't see any message bigger than 150, so probably we will never re-allocate.

**Comment by thaystg** [comment]
@pavelsavara could you please review again, now that it will be merged after your ts changes?

File: src/mono/wasm/runtime/src/mono/debug.ts
Comment (radical) [inline src/mono/wasm/runtime/src/mono/debug.ts:56]: inconsistent spacing before open parentheses.

Comment (thaystg) [reply src/mono/wasm/runtime/src/mono/debug.ts]: Thanks, fixed.

File: src/mono/wasm/runtime/src/mono/debug.ts
Comment (pavelsavara) [inline src/mono/wasm/runtime/src/mono/debug.ts:48]: This doesn't have to be exported

File: src/mono/wasm/runtime/src/mono/debug.ts
Comment (pavelsavara) [inline src/mono/wasm/runtime/src/mono/debug.ts:14]: the type could be Uint8Array

**Pull request merged**
